R/select_utilities.R

```diff
@@ -290,7 +290,7 @@
     dplyr::select(all_of(c(variable_column, select_column))) %>%
     dplyr::filter(stats::complete.cases(.)) %>%
     dplyr::filter({{ select_expr }}) %>%
-    dplyr::pull(all_of(variable_column)) %>%
+    dplyr::pull(!!rlang::sym(variable_column)) %>%
```

Comment:
I think you should leave this as is since we'll changing tidyselect in order to avoid having to use this sort of construction.

Reply:
will do!